README.md

```diff
@@ -60,22 +59,19 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Hello Microverse <a name="about-project"></a>
 
-**[your_project__name]** is a...
+> it the first traial hello microverse project
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
 
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://reactjs.org/">html</a></li>
```

Comment:
- [ ] This link is currently not representing what the user thought it was because it links to React's page instead of `html`. Kindly remove the link or look for resources that could fit like `https://html.com` or HTML resources on sites like `W3Schools`, `W3Cs`, etc.
This is to ensure that users get accurate information according to their needs. It's not a good user experience to hop on a link that leads to unexpected resources. :thinking: